Cesium3DTiles/src/BingMapsRasterOverlay.cpp

```diff
@@ -29,6 +29,13 @@ struct CreditAndCoverageAreas {
   Cesium3DTiles::Credit credit;
   std::vector<CoverageArea> coverageAreas;
 };
+
+struct SessionCacheItem {
+  std::string metadataUrl;
+  std::vector<std::byte> responseData;
+};
+
+std::vector<SessionCacheItem> sessionCache;
```

Comment:
Why not use `std::unordered_map<std::string, std::vector<std::byte>>` instead?

Reply:
That's a good point. I remember being told it's sometimes good to be semantically clear by naming the inner types instead of parameterizing unnamed types as in: `<std::string, std::vector<std::byte>>`. But I wasn't sure when that principle is applicable and when it's unnecessary, I'll change this to an unordered_map!

Reply:
Yeah that's true. Using `std::vector<SessionCacheItem>` is definitely much better than using `std::vector<std::pair<std::string, std::vector<std::byte>>>` for example. But since sessionCache is truly a map, it's certainly ok to use a map type here. The little we lose by being unable to name the key and value are more than made up for by the conceptual clarity of using an actual map, IMO.